app/controllers/data_access_controller.rb

```diff
@@ -115,6 +115,11 @@ def retrieval
           s['service_options']['number_processed'] = status['numberProcessed']
           s['service_options']['total_number'] = status['totalNumber']
           s['service_options']['download_urls'] = urls
+
+          if s['order_status'] == 'failed' && response_json['Exception'].nil? && !process_info.nil?
+            s['error_message'] = Array.wrap(process_info['message']).join("\n")
```

Comment:
@macrouch, @mightynimble, if there are multiple error messages, this does not actually allow them to show up on separate lines. If you have any suggestions to quickly change it so the messages can show up on separate lines (here or on the ko side), please let me know.